docs/releases.md

```diff
@@ -0,0 +1,65 @@
+# Release process
+
+The release process in broken up into 3 steps.
+
+1. Increment the version and add a new tag
+2. Modify the changelog
+3. Merge the release branch and create a "github release"
+
+## 1. Increment the version and add a new tag
+
+Run
+
+```shell
+$ yarn version
+```
+
+See [Semantic Versioning 2.0.0](https://semver.org/#semantic-versioning-200).
+
+- MAJOR version when you make incompatible API changes,
+- MINOR version when you add functionality in a backwards compatible manner, and
+- PATCH version when you make backwards compatible bug fixes.
+
+In most cases increment the MINOR version number only. The only change should be to package.json, and there should be a new tag on the commit prefixed with a "v".
+
+## 2. Modify the changelog
+
+Frontend-share is a consumed package, so it is important to emphasize changes that could affect anything that imports it. For this reason, the `CHANGELOG.md` file is used to highlight such changes. `CHANGELOG.md` is based our changelog off [keepachangelog](https://keepachangelog.com/en/1.0.0/_).
+
+### Steps to append to the changelog
+
+Immediately after a version commit & tag, run the follow command to create a draft list of changes relating to the new version.
+
+```shell
+$ gulp changelog
+```
+
+This command will generate a partial changelog and output it to the console. Copy the text and paste it above the previous version in `CHANGELOG.md`. The output is git logs between the current new version and the previous released version. It will be important to edit and trim down that list so only the user facing changes are clear, concise and obvious.
+
+### Edit the output
+
+1. Delete any commits that are not relevant for a consumer.
+2. Categorize the remaining important commits into various "types of changes" (defined below).
+3. Make sure to prefix anything that might break with "BREAKING CHANGE:"
+4. Refactor the messages as needed to ensure they can be understood by a consumer.
+
+#### Types of changes
+
+- `### Added` for new features.
```

Comment:
I suggest the "Types of changes" section could be replaced with a sentence that links to the "How do I make a good changelog" section of https://keepachangelog.com/en/1.0.0/#how. Something along the lines of `When writing the changelog, please follow the [Keep a Changelog advice](...)`.